src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -1260,6 +1261,7 @@ public void testDuplicateStudy() throws Exception {
         node2.setLoadFlowStatus(LoadFlowStatus.CONVERGED);
         node2.setLoadFlowResult(new LoadFlowResultImpl(true, Map.of("key_1", "metric_1", "key_2", "metric_2"), "logs"));
         node2.setSecurityAnalysisResultUuid(UUID.randomUUID());
+        //node2.setDynamicSimulationResultUuid(UUID.randomUUID());
```

Comment:
??

Reply:
comment is opened and move to the last. Done